src/EditorFeatures/CSharpTest/GenerateFromMembers/GenerateEqualsAndGetHashCodeFromMembers/GenerateEqualsAndGetHashCodeFromMembersTests.cs

```diff
@@ -1119,7 +1121,109 @@ public override bool Equals(object obj)
     }
 }",
 chosenSymbols: null,
-optionsCallback: options => options[0].Value = true,
+optionsCallback: options => EnableOption(options, GenerateOperatorsId),
+ignoreTrivia: false);
+        }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateEqualsAndGetHashCode)]
+        public async Task TestImplementIEquatable1()
```

Comment:
Better names for tests would be nice (mention that the interesting part is that it's a struct for example)